src/restapi/fCreateTicket.ts

```diff
@@ -1,34 +1,38 @@
 import {Request, Response} from 'express';
-import {AppCallResponse, AppCallRequest} from 'mattermost-redux/types/apps';
+import {AppCallRequest} from 'mattermost-redux/types/apps';
 
-import {newFormCallResponse, newOKCallResponse, newErrorCallResponseWithMessage} from '../utils/call_responses';
+import {newFormCallResponse, newErrorCallResponseWithMessage} from '../utils/call_responses';
 import {newCreateTicketForm} from '../forms';
 import {newApp} from '../app/app';
 
 // fOpenCreateTicketForm opens a new create ticket form
 export async function fOpenCreateTicketForm(req: Request, res: Response): Promise<void> {
-    const form = await newCreateTicketForm(req.body);
-    const callResponse = newFormCallResponse(form);
-    res.json(callResponse);
+    try {
+        const form = await newCreateTicketForm(req.body);
+        res.json(newFormCallResponse(form));
+    } catch (error) {
+        res.json(newErrorCallResponseWithMessage('Unable to open create ticket form: ' + error.message));
+    }
 }
 
 // fSubmitOrUpdateCreateTicketForm updates the create ticket form with new values or
 // submits the ticket if submit button is clicked
 export async function fSubmitOrUpdateCreateTicketForm(req: Request, res: Response): Promise<void> {
-    const form = await newCreateTicketForm(req.body);
-    const callResponse = newFormCallResponse(form);
-    res.json(callResponse);
+    try {
+        const form = await newCreateTicketForm(req.body);
+        res.json(newFormCallResponse(form));
+    } catch (error) {
+        res.json(newErrorCallResponseWithMessage('Unable to update create ticket form: ' + error.message));
+    }
 }
 
 // fSubmitOrUpdateCreateTicketSubmit creates a ticket
 export async function fSubmitOrUpdateCreateTicketSubmit(req: Request, res: Response): Promise<void> {
     const call: AppCallRequest = req.body;
-    let callResponse: AppCallResponse = newOKCallResponse();
     try {
         const app = newApp(call);
-        callResponse = await app.createTicketFromPost();
+        res.json(await app.createTicketFromPost());
```

Comment:
I have to say that I'm not a fan of this syntax. 1/5 having awaits in the middle of a multi-operation statement can be difficult to read/debug
```suggestion
        const callResponse = await app.createTicketFromPost()
        res.json(callResponse);
```

Reply:
done.  Any awaited call is resolved to a `callResponse` variable that is passed to `res.json`